test/python/test_nexus_multipath.py

```diff
@@ -46,12 +46,22 @@ def create_nexus(
     assert len(hdls["ms1"].pool_list().pools) == 1
     assert len(hdls["ms2"].pool_list().pools) == 1
 
+    return uri
+
+
+@pytest.fixture
+def create_nexus(create_nexus_no_destroy, mayastor_mod, nexus_name):
+    hdls = mayastor_mod
+    NEXUS_NAME = nexus_name
+    uri = create_nexus_no_destroy
     yield uri
     hdls["ms3"].nexus_destroy(NEXUS_NAME)
 
 
 @pytest.fixture
-def create_nexus_2(mayastor_mod, nexus_name, nexus_uuid, min_cntlid_2, resv_key_2):
+def create_nexus_2_no_destroy(
```

Comment:
are these "_no_destroy" so we can control the order of destruction?

Reply:
Yes.